tests/tests_default_wrapper.yml

```diff
@@ -10,6 +10,22 @@
             suffix: .yaml
           register: tempinventory
 
+        - name: Get os version of the control node
+          command: "hostnamectl"
+          register: __result
+
+        - name: Ensure required packages exist
+          package:
+            name:
+              - python3-libselinux
+              - policycoreutils
+            state: present
+          when: __result.stdout is
+                  search("Operating System. Red Hat Enterprise Linux 9.*")
+                or __result.stdout is
+                  search("Operating System. CentOS Linux 9.*")
+          delegate_to: localhost
+
```

Comment:
It isn't clear to me where the requirement for the selinux packages is coming from.  The error is coming from the task `name: Create static inventory from hostvars` which is just a `template` call using a simple file which does not use selinux directly.  The only thing I can think of is that the `template` module can set selinux policy on the file that it is managing - but then why doesn't every `template` call in every role have this problem?

Also - I think it would be better to use facts from localhost e.g. something like
```
- name: Get facts from localhost
  gather_facts:
  delegate_to: localhost

- name: Ensure packages ...
  when:
    - ansible_distribution in ["RedHat", "CentOS"]
    - ansible_distribution_major_version | int >= 9
  delegate_to: localhost
```
or something like that - not sure of the exact syntax